### PR TITLE
Join table

### PR DESCRIPTION
**Pull request by hopeforkoami**
### In this Branch, I:
  -  Update vets_id and visits_id column names.
  -  Add email column to owners table.
  -  Add data to owners and visits tables.
  -  Audit execution time performance of the given queries.
  -  Improve execution time performance.
  

### Queries Audit ScreenShots

**Query1 before** 

explain_query1

**Query1 After**

explain_query1_reduced

**Query2 before**
 
explain_query2

**Query2 After**
explain_query2_reduced

**Query3 before** 
explain_query3

**Query3 After**
explain_query3_reduced

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -44,4 +44,13 @@ CREATE TABLE visits (
...
+);
+
+ALTER TABLE visits RENAME COLUMN animals_id to animal_id;
+ALTER TABLE visits RENAME COLUMN vets_id to vet_id;
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+ALTER TABLE visits ADD COLUMN id serial primary key;
+
+CREATE INDEX ON visits(animal_id);
+CREATE INDEX ON visits(vet_id);
```
Comment (MahmoudBakr23) [inline schema.sql]: - [ ] Please add the missed `CREATE INDEX ON owners(email);`
_Don't forget to update the screenshots in the PR summary after that._ 😉

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello @hopeforkoami 😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on creating `indexes` for both `animal_id` and `vet_id`
- Well done adding the correct statements in the data, queries, and schema files
- Descriptive PR summary and README file

However, there is always room for improvement.

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (HAMMAS-SALEEM) [inline schema.sql]: - [ ] I guess you forgot to add the `CREATE INDEX ON owners(email);` query in this file with other Index queries which was suggested by the previous reviewer :smile: Kindly add this query here :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @hopeforkoami :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the requested changes, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 2]
```diff
@@ -85,4 +85,10 @@ SELECT COUNT(*) FROM visits
 
 SELECT species.name FROM visits
   JOIN animals ON animals.id = animals_id JOIN vets ON vets.id = vets_id JOIN species ON species.id = species_id
-  WHERE vets.name = 'Maisy Smith' GROUP BY species.name ORDER BY COUNT(species.name) DESC LIMIT 1;
\ No newline at end of file
+  WHERE vets.name = 'Maisy Smith' GROUP BY species.name ORDER BY COUNT(species.name) DESC LIMIT 1;
+
+
```
Comment (HAMMAS-SALEEM) [inline queries.sql:90]: **[Optional]**
- [ ] As you know that this is a pair-programming activity but I cannot see any commit from your partner's side :fearful: Kindly ask your partner to make some contributions to this pull request :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]

File: schema.sql (hunk 1 above)
Comment (hopeforkoami) [reply schema.sql]: please I forgot to push

File: queries.sql (hunk 2 above)
Comment (hopeforkoami) [reply queries.sql:90]: I'm alone my coding partner is absent, I send a request to the student success

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @hopeforkoami ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**